source/server/http/admin.h

```diff
@@ -31,6 +31,33 @@
 namespace Envoy {
 namespace Server {
 
+/**
+ * Implementation of Server::AdminStream.
+ */
+class AdminStreamImpl : public AdminStream {
+public:
+  AdminStreamImpl(Http::StreamDecoderFilterCallbacks& callbacks, Http::HeaderMap& request_headers,
+                  std::list<std::function<void()>>& on_destroy_callbacks)
+      : callbacks_(callbacks), request_headers_(request_headers),
+        on_destroy_callbacks_(on_destroy_callbacks){};
+  void setEndStreamOnComplete(bool end_stream) override { end_stream_on_complete_ = end_stream; }
+  void addOnDestroyCallback(std::function<void()> cb) override;
+  const Http::StreamDecoderFilterCallbacks& getDecoderFilterCallbacks() const override {
+    return callbacks_;
+  }
+  const Http::HeaderMap& getRequestHeaders() const override { return request_headers_; }
+  bool endStreamOnComplete() const override { return end_stream_on_complete_; }
+
+private:
+  // the callbacks member is a reference to a member of a ConnectionManagerImpl object. It is
```

Comment:
nit: `The callbacks`.  Please capitalize the first letter of each sentence here and later in this comment.